### PR TITLE
add some functions to MarkdownWriter class

### PR DESCRIPTION
**Pull request by younesdessia**
* **What kind of changes does this PR introduce?** 
- [ ] Bug fix
- [X] new features
- [ ] performance 
- [ ] docs update

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes:
- [X] No

* **Other information**:

### CODE REVIEW

**Review by Tanguylo — COMMENTED** [review]
Just change something in docstrings from other codes but don't increase min note please :)

File: code_pylint.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 UNWATCHED_ERRORS = ["fixme", "trailing-whitespace", "import-error"]
 
 MAX_ERROR_BY_TYPE = {
-    "wrong-spelling-in-docstring": 179,
+    "wrong-spelling-in-docstring": 180,
```
[the remark this answers is not in the document]
Comment (younesdessia) [reply code_pylint.py]: it's good, it's done !

File: dessia_common/tests.py [hunk 2]
```diff
@@ -371,7 +371,7 @@ def __init__(self, p_1: float, p_2: float, p_3: float, p_4: float, p_5: float, p
 
     @property
     def test_prop(self):
-        """ Factice property for some tests. """
+        """ Dummy property for some tests. """
```
Comment (Tanguylo) [inline dessia_common/tests.py:374]: ;)

**Review by Tanguylo — APPROVED** [review]

**Review by GhislainJ — CHANGES_REQUESTED** [review]
Just some small changes

**Review by GhislainJ — APPROVED** [review]

**Pull request merged**